gateway/project.go

```diff
@@ -71,6 +71,49 @@ func (g *Gateway) GetProject(ctx context.Context, projectId string) (*entity.Pro
 	return resp.Project, nil
 }
 
+func (g *Gateway) GetProjectByName(ctx context.Context, projectName string) (*entity.Project, error) {
+	gqlReq, err := g.NewRequestWithAuth(`
+		query ($projectName: String!) {
+			me {
+				projects(where: { name: { equals: $projectName } }) {
```

Comment:
Woah, didn't know you could do this

Reply:
How does this work. Just out of the box?

Reply:
I think because this 
<img width="357" alt="image" src="https://user-images.githubusercontent.com/587576/121736799-e465e200-caac-11eb-8473-40f6d73965d7.png">

Reply:
Oh its the me query. Thats really cool